scaife-viewer/scaife-viewer
# Refactor the version list template as a Vue component

#### What does this PR do?

Adds a new component to render the version list using Vue

#### Where should the reviewer start?

`CTSVersionList.vue`

#### How should this be manually tested?

- Browse to `/library/urn:cts:greekLit:tlg0012.tlg001/`
- Click the version's title to navigate to the table of contents
- Navigate back and click "Read (Greek)" to read the passage
- Version list should look and function identically to <https://scaife.perseus.org/library/urn:cts:greekLit:tlg0012.tlg001/>

#### Screenshot

![image](https://user-images.githubusercontent.com/629062/55757269-269cc000-5a19-11e9-9553-e7499f6d5fd1.png)
